dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -216,6 +215,23 @@ abstract class DLCWallet extends Wallet with AnyDLCHDWalletApi {
               } else Future.successful(withState)
             }
 
+            for {
+              updated <- updatedF
+
+              _ <- {
+                updated.state match {
+                  case DLCState.Claimed | DLCState.RemoteClaimed |
+                      DLCState.Refunded =>
+                    logger.info(
+                      s"Deleting unneeded DLC signatures for contract ${updated.contractIdOpt.get}")
+                    dlcSigsDAO.deleteByParamHash(updated.paramHash)
```

Comment:
might be worth sanity checking sigs and outcome are present in the db before wiping? Also is there any chance we want to put a timer on this happening (e.g. delete after 10 days or something like that?)